apps/generic_terminal/default_unix_utilities.csv

```diff
@@ -0,0 +1,75 @@
+awk,ark
+base64 -d,base sixty four decode
+base64,base sixty four
+cat,concat
+chmod -R,change mode recurse
+chmod,change mode
+chown -R,change owner recurse
+chown,change owner
+curl,curl
+cut,cut
+df -h,disk free human
+df,disk free
+dig,dick
```

Comment:
This might not go over well in an office heh

Reply:
True :D The benefits of working from home. I added this alternate voice command because I am a non-native speaker. I had trouble with accuracy with `dig`, but the reason is probably because I'm german and tend to pronounce consonants a little more hard than the usual english native speaker. I reverted this.